File: samples/wordnik-api-java/src/main/java/com/wordnik/client/model/FacetValue.java
package com.wordnik.client.model;

public class FacetValue {
  private String value;
  private long count;

  public String getValue() {
    return value;
  }
  public void setValue(String value) {
    this.value = value;
  }
  
  public long getCount() {
    return count;
  }
  
  public void setCount(long count) {
    this.count = count;
  }
}

File: samples/wordnik-api-scala/src/main/scala/com/wordnik/client/model/FacetValue.scala
package com.wordnik.client.model

import scala.reflect.BeanProperty

class FacetValue {
  @BeanProperty var value: String = _
  @BeanProperty var count: Long = 0L

  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class FacetValue {\n")
    sb.append("  value: ").append(value).append("\n")
    sb.append("  count: ").append(count).append("\n")
    sb.append("}\n")
    sb.toString
  }
}

